.github/workflows/cron-update-turbo.yml

```diff
@@ -0,0 +1,27 @@
+name: Cron Update Turbo
+
+on:
+  # Run every 4 hours https://crontab.guru/every-4-hours
+  schedule:
+    - cron: '0 */4 * * *'
```

Comment:
I'm thinking we might want to do this less often. The reason being that turbo bugs might not show up until we do a release and it could be hard to track down if we update turbo every day.

How about once per week?

Reply:
Done!